rpc/client.go

```diff
@@ -114,15 +127,19 @@ func (c *Client) writePump(logger log.TMLogger) {
 			}
 
 			if err := c.conn.WriteMessage(websocket.TextMessage, message); err != nil {
-				logger.Error("error writing message to websocket", "err", err)
+				if err.Error() != "io: read/write on closed pipe" {
+					logger.Error("error writing message to WebSocket", "err", err)
+				}
 				return
 			}
 
 			n := len(c.send)
 			for i := 0; i < n; i++ {
 				msg := <-c.send
 				if err := c.conn.WriteMessage(websocket.TextMessage, msg); err != nil {
-					logger.Error("error writing message to websocket", "err", err)
+					if err.Error() != "io: read/write on closed pipe" {
```

Comment:
Same here.

Reply:
`io: read/write on closed pipe` error occurs whenever the client closes the WebSocket connection. So it's not really an error so much as being informed that the dialogue is over.
Perhaps give an Info message, rather than an error, to record when the WebSocket connect closed?